test/controllers/api/v2/hosts_controller_test.rb

```diff
@@ -568,7 +568,9 @@ def set_remote_user_to(user)
         are set, authentication should succeed w/o valid session cookies" do
     Setting[:authorize_login_delegation] = true
     Setting[:authorize_login_delegation_api] = true
-    set_remote_user_to users(:admin)
+    apache_source = AuthSourceExternal.where(:name => 'apache_module').first_or_create
```

Comment:
You should use `auth_sources(:external)` or `FactoryBot.create(:auth_source_external)` here (and in the other test).